libnavui-androidauto/src/main/java/com/mapbox/androidauto/car/navigation/speedlimit/SpeedLimitWidget.kt

```diff
@@ -26,68 +28,87 @@ class SpeedLimitWidget(
     /**
      * The horizontal margin of the widget relative to the map.
      */
-    marginX: Float = 26f,
+    marginX: Float = 14f,
```

Comment:
It seems there is padding built into the images.

Is there a way to calculate the new image padding?

![Screen Shot 2022-09-06 at 1 25 54 PM](https://user-images.githubusercontent.com/3021882/188731507-ac43f780-65af-4f44-8143-ce27b4f21e95.png)

It will be difficult to align edge to edge with this, the margin of the compass and speed limit are after this? It may not look right on all screen resolutions

Reply:
I am not certain what the margin means yet. Would a margin of `0` look like this? Because that would be good.

![Screen Shot 2022-09-06 at 1 33 49 PM](https://user-images.githubusercontent.com/3021882/188732747-af51d16a-ab4b-48b3-9bad-40c29e398f8f.png)

Reply:
These margins don't affect how the bitmaps are drawn, only how they are positioned. In order to remove the padding I will have to remove the shadow, but it is there according to the [design](https://www.figma.com/file/6bXLQGCUaYb7DdJsNL3e0Z/DS-Navigation?node-id=4547%3A41985).

Reply:
This margin definition is difficult to use because the shadow does not have a clear edge, and the size of the shadow may change or even become a parameter. [RoadLabelOptions](https://github.com/mapbox/mapbox-navigation-android/blob/e3e78f48abb593b296f544ea93af49e594866011/libnavui-androidauto/src/main/java/com/mapbox/androidauto/car/navigation/roadlabel/RoadLabelOptions.kt#L10-L17) only let you specify colors 

![Screen Shot 2022-09-06 at 5 22 43 PM](https://user-images.githubusercontent.com/3021882/188762259-ae831159-f40f-4893-a7d6-40ea088a52dd.png)

^ Here's the design for reference. The purple rectangle defines the speed limit but not the shadow, are you sure the shadow is really coming from the Figma rendering?

IMO we could remove the margin parameters, or remove the shadow. It is not super important to define a margin, and there are many potential parameters here.

![Screen Shot 2022-09-06 at 5 54 48 PM](https://user-images.githubusercontent.com/3021882/188765352-5b986269-7405-4d40-9988-3ef2e67fc17a.png)

^ Honestly can barely see the shadow, so removing the shadow seems fine too.

Reply:
The rectangle in Figma doesn't include the shadow, the same happens with Android views (their width and height don't include the shadow). But when you draw on Canvas you have to add padding for the shadow, otherwise it won't be drawn. 
I would remove the margin parameters, because even with the current defaults widgets are not aligned with the action buttons: https://github.com/mapbox/1tap-android/issues/1543#issuecomment-1210374476. 
Also I updated the way the shadow is drawn, now it should look exactly like in Figma and be noticeable on the map.

Reply:
> I would remove the margin parameters

Let's do this. 

> the same happens with Android views

The caller gets to specify the shadow size in some way though, that way the caller can create alignment. So the shadow size could be a parameter. But these parameters would be better off in `SpeedLimitOptions` so they are protected from SEMVER (which can be added at any time, so we don't really need to do it now)